openapi.json

```diff
@@ -0,0 +1,9995 @@
+            "title": "register_type",
+            "type": "string"
+          },
+          "status": {
+            "description": "Current Status of Company / Master Account",
+            "enum": ["Active", "DQ Earmark", "Blocked", "Closed"],
+            "title": "status",
+            "type": "string"
+          },
+          "tags": { "items": { "type": "integer" }, "type": "array" }
+        },
+        "required": [
+          "company_name",
+          "compliance_risk_score",
+          "noga_code",
+          "register_id",
+          "register_type",
+          "status"
+        ],
+        "type": "object"
+      },
+      "memo": {
+        "example": {
+          "basic_accounts": [0, 0],
+          "cards": [6, 6],
+          "changed_by": "changed_by",
+          "changed_on": "2000-01-23T04:56:07+00:00",
+          "created_by": "created_by",
+          "created_on": "2000-01-23T04:56:07+00:00",
+          "desccription": "desccription",
+          "id": 1,
+          "master_accounts": [5, 5],
+          "owner": [5, 5],
+          "persons": [2, 2],
+          "subject": "subject",
+          "tags": [7, 7],
+          "type": "Billing"
+        },
+        "properties": {
+          "basic_accounts": { "items": { "type": "integer" }, "type": "array" },
+          "cards": { "items": { "type": "integer" }, "type": "array" },
+          "changed_by": {
+            "description": "User name/email of the user who last changed the record via API",
+            "readOnly": true,
+            "title": "changed_by",
+            "type": "string"
+          },
+          "changed_on": {
+            "description": "Timestamp on which the record was last changed, automatically filled by the API",
+            "format": "date-time",
+            "readOnly": true,
+            "title": "changed_on",
+            "type": "string"
+          },
+          "created_by": {
+            "description": "User name/email of the user who inserted the record via API",
+            "readOnly": true,
+            "title": "created_by",
+            "type": "string"
+          },
+          "created_on": {
+            "description": "Timestamp on which the record was created, automatically filled by the API",
+            "format": "date-time",
+            "readOnly": true,
+            "title": "created_on",
+            "type": "string"
+          },
+          "desccription": {
+            "description": "Memo Description / Details",
+            "nullable": true,
+            "title": "desccription",
+            "type": "string",
+            "x-nullable": true
+          },
+          "id": {
+            "description": "to be documented",
+            "readOnly": true,
+            "title": "id",
+            "type": "integer"
+          },
+          "master_accounts": {
+            "items": { "type": "integer" },
+            "type": "array"
+          },
+          "owner": { "items": { "type": "integer" }, "type": "array" },
```

Comment:
Which type is this referring to? `person`, `master_account`, other?

Reply:
User, I can extend the compiler such that it returns:
        owner:
          items:
            description: User
            type: integer
          type: array
would that help?

Reply:
Thanks! Since owner always points to user id, we'll just hardcode it.